src/invariant/InvariantManager.h

```diff
@@ -50,10 +51,20 @@ class InvariantManager
                           LedgerTxnDelta const& ltxDelta,
                           std::vector<ContractEvent> const& events) = 0;
 
+    virtual void checkOnLedgerCommit(
+        SearchableSnapshotConstPtr lclLiveState,
+        SearchableHotArchiveSnapshotConstPtr lclHotArchiveState,
+        std::vector<LedgerEntry> const& evictedFromLive,
```

Comment:
I don't think `evictedFromLive` vs `deletedKeysFromLive` is super clear. Maybe rename `persitentEvictedFromLive` and `tempAndTTLEvictedFromLive` or something. Both are "deleted" from the live BucketList so it's a little confusing as stated currently.